robottelo/datafactory.py

```diff
@@ -111,7 +111,6 @@ def invalid_values_list(interface=None):
 
     :param str interface: Interface name (one of api/cli/ui).
     :return: Returns the invalid values list
-    :rtype: list
```

Comment:
```
more than one target found for cross-reference u'list': robottelo.cli.docker.DockerRegistry.list, robottelo.cli.docker.DockerImage.list, robottelo.cli.docker.DockerTag.list, robottelo.cli.lifecycleenvironment.LifecycleEnvironment.list, robottelo.cli.docker.DockerContainer.list, robottelo.cli.base.Base.list
```
That's the weirdest one. sphinx should check built-ins first, and only if corresponding built-in wasn't found it should check other modules. No idea why it's ignoring built-in `list` type and searches for it somewhere else, that sounds like a sphinx bug for me. Googled a bit and found similar case which was considered as a bug.
Anyway, to make the travis happy and not to leave the single warning, i'd like to remove this line. `:return:` block already says that's a list, and the rest functions from datafactory also describe return type in `:return:` section, without any `:rtype:`